multinet/db.py

```diff
@@ -82,6 +84,7 @@ def workspace_mapping_collection() -> StandardCollection:
     return sysdb.collection("workspace_mapping")
 
 
+@lru_cache()
 def workspace_mapping(name: str) -> Union[Dict[str, str], None]:
```

Comment:
Is there a huge benefit to adding a cache here since it just calls out to another function you have cached and does a quick look up?

Reply:
Running the `find` command in this function makes a call to the database server, so caching the documents themselves here allows us to skip that call entirely.

Reply:
Roger that